src/admin/BulkUserCreationPage.tsx

```diff
@@ -32,8 +32,8 @@ const BulkUserCreationPage: FC = () => {
     }),
     onSubmit: ({ role, emailsString }) => {
       const emails = getEmails(emailsString);
-      // TODO: support ESTONIAN_ID authenticaiton method
-      const command: CreateUserCommand[] = emails.map((email) => ({
+      // TODO: support ESTONIAN_ID authentication method
```

Comment:
Is this still relevant?  What needs to be added?

Reply:
The bulk user creation page currently only works for the `MAGIC_LINK` method indeed, as it hardcodes the method and has email-specific copy.

When we see a use case for the page with other methods, we can mimic what's being done in this PR for the add test for identifier page and change copy and passed authentication method depending on the config.